test/lib/utils/config/definition.js

```diff
@@ -21,6 +21,11 @@ t.test('basic definition', async t => {
     default: 'some default value',
     defaultDescription: '"some default value"',
     type: [Number, String],
+    hint: 'key',
+    usage: [
+      '--key <key>',
```

Comment:
We will want to clean this up when we get to a command that has this kind of config declaration, until then it's fine imo.